collectd/files/collectd_logrotate.conf

```diff
@@ -0,0 +1,4 @@
+/var/log/{{ service_name }}.log {
```

Comment:
you may want to use these options:
copytruncate
compress         
delaycompress    

and optionally these ones as well:
minsize 20M  
maxsize 50M